cmd/wavelet/cli.go

```diff
@@ -43,32 +44,44 @@ const (
 )
 
 type CLI struct {
-	app    *cli.App
-	rl     *readline.Instance
-	client *skademlia.Client
-	ledger *wavelet.Ledger
+	app *cli.App
+	rl  *readline.Instance
+	// client *skademlia.Client
```

Comment:
why its commented instead of being removed?
and why it was commented in the first place?

Reply:
`*skademlia.Client` was moved to `package server` to split the CLI logic from these (and also to make later TUI development easier)